pandas/tests/util/test_assert_almost_equal.py

```diff
@@ -454,6 +455,19 @@ def test_assert_almost_equal_shape_mismatch_override():
         tm.assert_almost_equal(np.array([1, 2]), np.array([3, 4, 5]), obj="Index")
 
 
+@pytest.mark.parametrize(
+    "obj",
+    [
+        MockNumpyLikeArray(np.ndarray((2,) * 1)),
+        MockNumpyLikeArray(np.array([])),
```

Comment:
why isnt this considered iterable?

Reply:
Hi, when I try iterating them:

```
TypeError                                 Traceback (most recent call last)
----> 1 next(np.ndarray((2,) * 1))

TypeError: 'numpy.ndarray' object is not an iterator

TypeError                                 Traceback (most recent call last)
Cell In[5], line 1
----> 1 next(np.array([]))

TypeError: 'numpy.ndarray' object is not an iterator
```

Reply:
my mistake, they're both iterable:

```
iter(np.ndarray((2,) * 1))
<iterator at 0x202471fbdf0>

iter(np.array([]))
<iterator at 0x20247252400>

iter(np.array(2))
Cell In[37], line 1
----> 1 iter(np.array(2))

TypeError: iteration over a 0-d array
```
so those two can be removed

Reply:
fixed, thank you